comp/otelcol/otlp/testutil/testutil.go

```diff
@@ -133,3 +147,346 @@ func fillLogTwo(log plog.LogRecord) {
 
 	log.Body().SetStr("something happened")
 }
+
+var (
```

Comment:
Everything below is copied from https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/internal/testutil/test_utils.go